test/tests/api/emails.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+describe('/POST emails', ()=> {
+  it('should save message to DB', (done)=> {
+    let message = { body: 'i love you'};
+    request
+      .post('/emails')
+      .send(message)
+      .expect(201)
+      .end((err,res)=> {
+        if(err) return done(err);
+        res.body.should.be.a('object');
+        res.body.should.have.property('body', message.body);
+        res.body.should.have.property('_id');
+      })
+    done();
```

Comment:
Remove this call to `done`, else the test doesn't actually run.
We only expect `done` to be invoked within the `end(...)` callback.

Reply:
To clarify, move `done()` right after line 14